.buildkite/setup-env.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# Install Go
+export PATH=$PATH:/usr/local/go/bin
+
+# Install kind
+curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.11.1/kind-linux-amd64
+chmod +x ./kind
+mv ./kind /usr/local/bin/kind
+
+# Install Docker
+bash scripts/install-docker.sh
+
+# Delete dangling clusters
+kind delete clusters --all
+
+# Install kubectl.
+curl -LO https://dl.k8s.io/release/v1.27.3/bin/linux/amd64/kubectl
+curl -LO "https://dl.k8s.io/release/v1.27.3/bin/linux/amd64/kubectl.sha256"
+echo "$(cat kubectl.sha256)  kubectl" | sha256sum --check
+install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
+
+# Create the cluster
+time kind create cluster --wait 120s --config tests/framework/config/kind-config-buildkite.yml
```

Comment:
Why do we need to create a Kind cluster (L24 - L56) here without running any tests?

Reply:
We don't need it anymore, this is just from the initial test.  I have a followup PR which will remove it

Reply:
I just removed it in this PR.